demo/test.html

```diff
@@ -101,8 +101,9 @@ <h3>Events</h3>
 
         Radar.initialize(publishableKey);
 
-        Radar.trackOnce(function(err, { location, user, events, response }) {
-          console.log(err, location, user, events);
+        Radar.trackOnce(function(err, { status, location, user, events }, response) {
+
+          console.log({ err, location, user, events, response });
```

Comment:
remove this?

Reply:
This is on the demo page, I think it's fine to leave it.